rpc/core/net.go

```diff
@@ -95,6 +95,20 @@ func (env *Environment) Genesis(ctx *rpctypes.Context) (*ctypes.ResultGenesis, e
 	return &ctypes.ResultGenesis{Genesis: env.GenDoc}, nil
 }
 
+func (env *Environment) GenesisChunked(ctx *rpctypes.Context, chunkID uint) (*ctypes.ResultGenesisChunk, error) {
+	id := int(chunkID)
```

Comment:
Do we want to add a check here that we have initialised the chunks. Will this fail acceptably if we haven't initialised the chunks?

Reply:
I added the check that if we give an ID that is out of bounds we'll generate an error, so it's "safe".